tests/integration/test_headers.py

```diff
@@ -20,6 +20,7 @@
 _EXPECTED_ACCESS_MODE = "user-restricted"
 
 _SPECIALTY_REF_DATA_URL = "/FHIR/STU3/CodeSystem/SPECIALTY"
+_SEARCH_SERVICE_REQUEST_R4_URL = "/FHIR/R4/ServiceRequest"
```

Comment:
Why was this added?

Reply:
Ah was this meant to be `R4/HealthcareService`  `A035 - Search for healthcare services`  instead?

Reply:
Ah I see now, this was meant to test an operation outcome returned on 401, but maybe we need to use another r4 endpoint in here, e.g. the a035 I mentioned above